osbrain/address.py

```diff
@@ -428,10 +440,12 @@ def __eq__(self, other):
 
     def twin(self):
         """
+        Get the twin channel of the current one.
+
         Returns
         -------
         AgentChannel
-            The twin channel of the current one.
+            The twin channel.
```

Comment:
Maybe instead of "Get the twin..." we could say "Return the twin..." and remove the "Returns" section from the docstring. Would apply to other docstrings above this one.

The idea Guillermo proposed was to only use a short description for simple functions/methods (i.e.: no parameters, single return).

On the other hand we lose the return type, but we could use typing for that (after all, the idea is to use typing everywhere for version 1.0). To specify the return type:

```python
def function_or_method(params) -> ReturnType:
    pass
```

What do you think?

Reply:
I like having the return type in the docs. Will sphinx be able to read typing and generate similar documentation to what we have right now?

Reply:
Yes, we probably should look at this again when we move to typing.

I'll keep it as it is (even if it's redundant), and we'll clean it up later if it's OK with you.

Reply:
Yeah, ok for me. Add an issue though, to try typing/Sphinx/napoleon and see if it works well or as expected.

Reply:
By the way, remember to open an issue for this. :wink: